cypress/e2e/tests/pages/manager/cluster-provisioning-amazon-ec2-rke2.spec.ts

```diff
@@ -268,6 +266,116 @@ describe('Deploy RKE2 cluster using node driver on Amazon EC2', { testIsolation:
     clusterDetails.snapshotsList().checkSnapshotExist(`on-demand-${ this.rke2Ec2ClusterName }`);
   });
 
+  it('can scale up a machine pool', function() {
+    // testing https://github.com/rancher/dashboard/issues/13285
+    const clusterDetails = new ClusterManagerDetailRke2AmazonEc2PagePo(undefined, this.rke2Ec2ClusterName);
+
+    ClusterManagerListPagePo.navTo();
+    clusterList.waitForPage();
+
+    // Navigate to cluster details page > machine pools
+    clusterList.list().name(this.rke2Ec2ClusterName).click();
+    clusterDetails.waitForPage(null, 'machine-pools');
+    clusterDetails.resourceDetail().title().should('contain', this.rke2Ec2ClusterName);
+
+    // Verify scaling buttons are present in the machine pools section
+    clusterDetails.poolsList('machine').resourceTable().sortableTable().groupByButtons(1)
+      .click();
+
+    // Check for scale up button (it should be enabled)
+    clusterDetails.poolsList('machine').scaleUpButton(`${ this.rke2Ec2ClusterName }-pool1`)
+      .should('be.visible')
+      .and('be.enabled');
+
+    // Hover scale up button - tooltip should read "Scale Pool Up"
+    clusterDetails.poolsList('machine').scaleButtonTooltip(`${ this.rke2Ec2ClusterName }-pool1`, 'plus')
+      .waitForTooltipWithText('Scale Pool Up');
+    clusterDetails.poolsList('machine').scaleButtonTooltip(`${ this.rke2Ec2ClusterName }-pool1`, 'plus')
+      .hideTooltip();
+
+    // Check for scale down button (it should be disabled initially)
+    clusterDetails.poolsList('machine').scaleDownButton(`${ this.rke2Ec2ClusterName }-pool1`)
+      .should('be.visible')
+      .and('be.disabled');
+
+    // Hover scale down button - tooltip should read "Scale Pool Down"
+    clusterDetails.poolsList('machine').scaleButtonTooltip(`${ this.rke2Ec2ClusterName }-pool1`, 'minus')
+      .waitForTooltipWithText('Scale Pool Down');
+    clusterDetails.poolsList('machine').scaleButtonTooltip(`${ this.rke2Ec2ClusterName }-pool1`, 'minus')
+      .hideTooltip();
+
+    cy.intercept('PUT', ` /v1/provisioning.cattle.io.clusters/fleet-default/${ this.rke2Ec2ClusterName }`).as('scaleUpMachineDeployment');
+    // Scale up the machine pool
+    clusterDetails.poolsList('machine').scaleUpButton(`${ this.rke2Ec2ClusterName }-pool1`)
+      .click();
+
+    cy.wait('@scaleUpMachineDeployment').its('response.statusCode').should('eq', 200);
+
+    // Verify the machine pool is scaled up to 2
+    clusterDetails.poolsList('machine').machinePoolCount(`${ this.rke2Ec2ClusterName }-pool1`, /^2$/, { timeout: 700000 });
```

Comment:
700,000ms is a very long timeout. If this is required, we might want to define a `VERY_LONG_TIMEOUT_OPT` constant rather than hardcoding numbers, just to keep it consistent.